deis-logger-beta1-test/manifests/deis-logger-fluentd-daemon.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: deis-logger-fluentd
+  namespace: deis
+  labels:
+    heritage: deis
+spec:
+  template:
+    metadata:
+      name: deis-logger-fluentd
+      labels:
+        heritage: deis
+    spec:
+      serviceAccount: deis-logger-fluentd
+      containers:
+      - name: deis-logger-fluentd
+        image: quay.io/deisci/fluentd:git-f537de4
+        imagePullPolicy: Always
```

Comment:
If we're pulling an immutable tag, this should change to `IfNotPresent`.  Looks like `workflow-beta1-test` has that wrong in a couple spots (but right in most spots).  I'll PR that separately, but this should be fixed here.